virtcontainers/store/filesystem_backend.go

```diff
@@ -65,15 +66,33 @@ const VMPathSuffix = "vm"
 
 // ConfigStoragePath is the sandbox configuration directory.
 // It will contain one config.json file for each created sandbox.
-var ConfigStoragePath = filepath.Join("/var/lib", StoragePathSuffix, SandboxPathSuffix)
+var ConfigStoragePath = func() string {
```

Comment:
I did it this way, so that in the unit tests I could set a different function to modify the return - sort of a mocking the function call. Is there a better way of doing that in unit tests?!

Reply:
@gabibeyer Didnt realise you were using them in the unit tests, we have done this differently in the past, but this looks good to me as well. Can you just add a comment here, that the function was declared this way for mocking in the unit tests for future reference.